scripts/build/build-releases.sh

```diff
@@ -11,46 +11,19 @@
 
```

Comment:
@halim-lee We can get rid of this script and all logic around 'multiple releases' since we'll only need to build a single release. This was something ported from OLO/RCO (which allowed us to manage their images in Docker Hub and refresh them from time to time). But with the images in icr.io and being referenced using SHA values in CSV, we'll have to do a new release.

Suggest to change build-release.sh to build-operator.sh. Thank you

Reply:
Yep removed them and testing the scripts!